src/Components/CreateImageWizardV2/steps/ImageOutput/ReleaseSelect.tsx

```diff
@@ -0,0 +1,117 @@
+import React, {
+  Dispatch,
+  ReactElement,
+  SetStateAction,
+  useRef,
+  useState,
+} from 'react';
+
+import {
+  Select,
+  SelectOption,
+  SelectList,
+  MenuToggle,
+  FormGroup,
+} from '@patternfly/react-core';
+
+import { RELEASES } from '../../../../constants';
+import { Distributions } from '../../../../store/imageBuilderApi';
+import isRhel from '../../../../Utilities/isRhel';
+
+type ReleaseSelectType = {
+  setRelease: Dispatch<SetStateAction<string>>;
```

Comment:
Can this be like this?
```suggestion
  setRelease: Dispatch<SetStateAction<Distributions>>;
```